[v13] Document DynamoDB backend billing_mode option (#29359)
gravitational/teleport: docs/pages/reference/backends.mdx

    # auto_scaling is used to optionally enable (and define settings for) auto scaling.
    # default: false
    auto_scaling:  [true|false]

>     # Enables either On-Demand or Provisioned billing for the DynamoDB table. Set when Teleport creates the table.

Reviewer: Doesn't look like this was updated here?
```suggestion
    # Enables either Pay-Per-Request or Provisioned billing for the DynamoDB table. Set when Teleport creates the table.
```

>     # Enables either On-Demand or Provisioned billing for the DynamoDB table. Set when Teleport creates the table.

Author: ah, there were two that needed fixing and I missed this one :facepalm:

}
```

<Admonition
  type="warning"
  title="pay_per_request with autoscaling"
>

>   If `pay_per_request` is set to true, the settings for read/write capacity and auto scaling will be ignored.

Reviewer 2: It's not a Boolean, so it can't be set to true. Please fix this.

Reviewer: Looks like the [original PR](https://github.com/gravitational/teleport/pull/28672) has the same issue so @lxea please send a follow up to fix it in master as well.

Reviewer 2: Yeah, sorry I missed the original. I approved with a few suggestions before these changes were made and a re-review was never requested.
